subsys/net/l2/openthread/openthread.c

```diff
@@ -334,9 +355,31 @@ static int openthread_init(struct net_if *iface)
 	return 0;
 }
 
+void openthread_joiner_autostart(struct net_if *iface)
+{
+	struct openthread_context *ot_context = net_if_l2_data(iface);
+	otError error;
+
+	error = otJoinerStart(ot_context->instance, CONFIG_OPENTHREAD_JOINER_PSKD, NULL, PACKAGE_NAME,
+			CONFIG_OPENTHREAD_PLATFORM_INFO, PACKAGE_VERSION, NULL, &ot_joiner_start_handler, ot_context);
```

Comment:
This looks like a line that is over 80 chars long, I wonder why checkpatch has not complained about it.